apps/gitness/src/components-v2/breadcrumbs/breadcrumbs.tsx

```diff
@@ -1,56 +1,60 @@
-import { Link, useMatches } from 'react-router-dom'
+import { UIMatch } from 'react-router-dom'
 
-import { Breadcrumb, Separator, Sidebar, Topbar, useSidebar } from '@harnessio/ui/components'
+import { Breadcrumb, Separator, Sidebar, Topbar } from '@harnessio/ui/components'
+import { useRouterContext, useTheme } from '@harnessio/ui/context'
 import { cn } from '@harnessio/ui/utils'
 
-import { useThemeStore } from '../../framework/context/ThemeContext'
-import { useIsMFE } from '../../framework/hooks/useIsMFE'
 import { CustomHandle } from '../../framework/routing/types'
 
-function Breadcrumbs() {
-  const matches = useMatches()
-  const { isMobile } = useSidebar()
-  const matchesWithBreadcrumb = matches.filter(match => (match.handle as CustomHandle)?.breadcrumb)
-  const isMFE = useIsMFE()
-  const { isInset } = useThemeStore()
+export interface BreadcrumbsProps {
+  breadcrumbs: UIMatch<unknown, CustomHandle>[]
```

Comment:
is `UIMatch` available on router v5? if not, this will break in the MFE

Reply:
Dang, looks like it isn't